### ISSUE TITLE
SOAP: getDataObjectXXX methods not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bmor1/bimserver#169]
```
In order to build a client I tested the available SOAP methods. When I try to 
fetch DataObjects from the server (eg. Service#getDataObjectsByType), a list 
with the expected number of DataObjects is returned. But unfortunately the 
DataObjects don't contain any values. Is this a feature or a bug?
```

Original issue reported on code.google.com by `hlgts...@googlemail.com` on 13 Dec 2010 at 1:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `berlo...@gmail.com` on 13 Dec 2010 at 1:27
- Added labels: **Milestone-Release1.0**

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in the 1.0 branch in SVN, RC5 will be made this week
```

Original comment by `ru...@logic-labs.nl` on 15 Dec 2010 at 1:25
- Changed state: **Fixed**
